Gruntfile.js

```diff
@@ -86,7 +86,7 @@ module.exports = function( grunt ) {
 						loops: false
 					},
 					banner: "/*! Sizzle v<%= pkg.version %> | (c) " +
-						"jQuery Foundation, Inc. | jquery.org/license */",
+						"JS Foundation, Inc. | js.foundation */",
```

Comment:
This should really read the same as the other copyright statements like

`JS Foundation and other contributors | js.foundation`